internal/cmd/operator-sdk/bundle/validate/validate.go

```diff
@@ -153,6 +174,21 @@ func (c bundleValidateCmd) run(logger *log.Entry, bundleRaw string) (res *intern
 	results = runOptionalValidators(bundle, c.selector, c.optionalValues)
 	res.AddManifestResults(results...)
 
+	// TODO: (zeus) consider making this a runExternalValidators method similar
+	// to the optional one above
```

Comment:
Nit: Is this `TODO` still something you think should be done?

It might be nice to have this be a function since it has been done before with the `runOptionalValidators` function.

Reply:
@everettraven for this PR, I wouldn't make this change yet. I am anticipating we might be refactoring things in the future. There is a `RunExternalValidators` that is getting called.